spec/controllers/editorial/news_controller_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'rails_helper'
+
+RSpec.describe Editorial::NewsController, type: :controller do
```

Comment:
My take: I prefer to have minimal controller specs (or even skip them altogether) & write proper integration tests with capybara API. Controller specs don't test the full stack (routing, etc) and don't really test the full user path ...